types/node/globals.d.ts

```diff
@@ -331,6 +332,11 @@ interface AbortSignal {
      * Returns true if this AbortSignal's AbortController has signaled to abort, and false otherwise.
      */
     readonly aborted: boolean;
+
+    /**
+     * Invoking this method will set this object's AbortSignal's aborted flag and signal to any observers that the associated activity is to be aborted.
+     */
+    abort(): void;
```

Comment:
I think this is being treated as an instance method which I'm not sure is correct.

There's nothing on the [node docs](https://nodejs.org/dist/latest-v16.x/docs/api/globals.html#globals_class_abortsignal) for an `abort()` instance method on AbortSignals, only a static `AbortSignal.abort()` method.  There's also nothing on the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) for this method and it's not in the [node.js source for AbortSignal](https://github.com/nodejs/node/blob/e46c680bf2b211bbd52cf959ca17ee98c7f657f5/lib/internal/abort_controller.js#L50-L70) either.

TS compilation with `@types/node@15.12.0` fails with:

```
 Argument of type 'import("/path/to/node_modules/abort-controller/dist/abort-controller").AbortSignal' is not assignable to parameter of type 'AbortSignal'.
  Property 'abort' is missing in type 'import("/path/to/node_modules/abort-controller/dist/abort-controller").AbortSignal' but required in type 'AbortSignal'.

  node_modules/@types/node/globals.d.ts:339:5
    339     abort(): void;
            ~~~~~~~~~~~~~~
    'abort' is declared here.
```

Reply:
this PR was already merged. you might wanna open an issue or better create a PR.

Reply:
Ah yea I read the changelog wrong, I will file a PR asap.

Reply:
Actually I cannot make it static as this will conflict with the lib dom typings, I will remove the method entirely for now.

Reply:
The PR is [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53591) for anyone with power to approve.